backend/src/bundles/auth/auth.service.ts

```diff
@@ -80,6 +80,12 @@ class AuthService {
 
         return user.toObject();
     }
+
+    public async findByToken(
+        token: string,
+    ): Promise<UserFindResponseDto | undefined> {
+        return await this.userService.findByToken(token);
```

Comment:
You don't need return await (http://hassansin.github.io/Why-return-await-Is-a-Bad-Idea)